mappings/net/minecraft/server/command/SpreadPlayersCommand.mapping

```diff
@@ -2,16 +2,27 @@ CLASS tk net/minecraft/server/command/SpreadPlayersCommand
 	CLASS tk$a Pile
 		FIELD a x D
 		FIELD b z D
+		METHOD a normalize ()V
```

Comment:
What's weird about this method and `absolute` is that it uses the distance from (x, z) = (0, 0) rather than the area center. This seems like a Mojang bug...

Reply:
Yeah, I wasn't really sure what's the purpose of these methods either.
When talking about vector arithmetic, this calculates the absolute value of the vector, tho, and noralize normalizes it/ makes it a length of one, which is why I named them these.
I agree though, using the center of the spreading as origin would make more sense here.